packages/lib/services/joplinCloudUtils.ts

```diff
@@ -0,0 +1,104 @@
+import { Reducer } from 'react';
+import Setting from '../models/Setting';
+import { ApplicationPlatform, ApplicationType } from '../types';
+import shim from '../shim';
+import { _ } from '../locale';
+import { reg } from '../registry';
+
+type Events = 'LINK_USED' | 'COMPLETED';
+
+type IntitialValues = {
+	className: 'text' | 'bold';
+	message: string;
+	next: Events;
+	active: Events | 'INITIAL';
+};
+
+export const intitialValues: IntitialValues = {
+	className: 'text',
+	message: _('Waiting for authorisation...'),
```

Comment:
I'm not sure translation would be working for this because it can be called before the locales are loaded. One way around this is to convert the message to a function `message: () => _(....)` or any other way that ensures `_(....)` is called only when needed and not in advance, like there.